contracts/sdk/bubblegum/idl/bubblegum.json

```diff
@@ -893,6 +893,12 @@
           },
           {
             "name": "DecompressV1"
+          },
```

Comment:
Note: I haven't changed bubblegum, I just re-ran `node soltia.js` with the latest on main